src/env.json

```diff
@@ -9,5 +9,6 @@
     "APP_BASE_URL_TOKEN": "",
     "APP_NAME": "SUNBIRD",
     "MODE": "standalone",
-    "DEVICE_REGISTRY_URL": ""
+    "DEVICE_REGISTRY_URL": "",
+    "whiteListUrls": ["https://www.youtube.com/", "https://s.ytimg.com", "https://www.google.com"]
```

Comment:
Blob url for Player is not listed. Please add that as well.
https://www.google.com is invalid url for whitelisting

Reply:
blob url will change depending on env. We cant add it here. That should be added at build time.